nodecore/api-calls-examples/nodecore-pending-transactions-webpage/src/Pages/PendingTransactions/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Table } from 'antd';
+import { getPendingTransactions } from '../../api/pendingTransactions';
+import { IPendingTransactions } from '../../models/IPendingTransactions';
+import { EXPLORER_TX_SEARCH_URL } from '../../constants';
+import { ITransactionWithFeePerByte } from '../../models/ITransaction';
+import { SortOrder } from 'antd/lib/table/interface';
+import { ITableFilter } from '../../models/ITableFilter';
```

Comment:
can we do something about these ../../ imports?

Reply:
yes we can. done